channel.go

```diff
@@ -181,13 +181,13 @@ const localTTL = 60
 
 // Channel is our struct for json and db representations of our channel
 type Channel struct {
-	OrgID       OrgID       `json:"org_id"        db:"org_id"`
-	ID          ChannelID   `json:"id"            db:"id"`
-	UUID        ChannelUUID `json:"uuid"          db:"uuid"`
-	ChannelType ChannelType `json:"channel_type"  db:"channel_type"`
-	Address     string      `json:"address"       db:"address"`
-	Country     string      `json:"country"       db:"country"`
-	Config      string      `json:"config"        db:"config"`
+	OrgID       OrgID          `json:"org_id"        db:"org_id"`
+	ID          ChannelID      `json:"id"            db:"id"`
+	UUID        ChannelUUID    `json:"uuid"          db:"uuid"`
+	ChannelType ChannelType    `json:"channel_type"  db:"channel_type"`
+	Address     sql.NullString `json:"address"       db:"address"`
+	Country     sql.NullString `json:"country"       db:"country"`
+	Config      sql.NullString `json:"config"        db:"config"`
```

Comment:
Not so related to this PR, but might be simpler if this was a map and the JSON parsing happened during the DB fetch. It's kind weird that the JSON serialized version of a channel will have serialized JSON as a field